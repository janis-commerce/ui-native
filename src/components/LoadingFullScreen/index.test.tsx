// import 'react-native';
// import React from 'react';
// import {create} from 'react-test-renderer';
// import Text from './index';

// describe('Text component', () => {
// 	describe('return error because children is undefined', () => {
// 		it('undefined children', () => {
// 			const {toJSON} = create(<Text />);
// 			expect(toJSON()).toBeNull();
// 		});
// 	});

// 	describe('render correct', () => {
// 		it('match snapshot', () => {
// 			const {toJSON} = create(<Text>Janis Picking</Text>);
// 			expect(toJSON()).toMatchSnapshot();
// 		});
// 	});
// });
