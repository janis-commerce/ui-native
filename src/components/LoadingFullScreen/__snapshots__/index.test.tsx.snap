// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingFullScreen component render correct render only the loading 1`] = `
<Modal
  animationType="fade"
  hardwareAccelerated={false}
  testID="loading modal"
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(255, 255, 255, 0.75)",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 64,
          "justifyContent": "center",
          "position": "relative",
          "width": 64,
        }
      }
    >
      <View
        collapsable={false}
        style={
          Object {
            "borderBottomColor": "#2979FF",
            "borderLeftColor": "#2979FF",
            "borderRadius": 32,
            "borderRightColor": "#2979FF",
            "borderTopColor": "#D0D3E3",
            "borderWidth": 3.5,
            "height": 64,
            "position": "absolute",
            "transform": Array [
              Object {
                "rotateZ": "0deg",
              },
            ],
            "width": 64,
          }
        }
      />
      <View>
        <View
          height={25}
          width={36}
        >
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight={25}
            bbWidth={36}
            fill="none"
            focusable={false}
            height={25}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "flex": 0,
                  "height": 25,
                  "width": 36,
                },
              ]
            }
            vbHeight={47}
            vbWidth={65}
            width={36}
            xmlns="http://www.w3.org/2000/svg"
          >
            <RNSVGGroup
              fill={null}
              propList={
                Array [
                  "fill",
                ]
              }
            >
              <RNSVGPath
                d="M50.1841 16.9578C46.4571 16.9578 43.6747 17.9474 41.1548 20.1724C40.9028 20.4012 40.6498 20.6386 40.3915 20.8812C40.1384 21.1217 39.8938 21.3622 39.6534 21.5974C39.6482 21.6027 39.6113 21.6388 39.607 21.6441L24.0672 38.2278C22.6459 39.7027 21.076 40.5842 20.6353 40.8152L20.455 40.9063C20.4466 40.9106 20.4381 40.9148 20.4297 40.919C18.9747 41.6194 17.5387 41.9605 16.0395 41.9605C9.66072 41.9605 4.47234 36.9045 4.47234 30.6882C4.47234 27.7512 5.7291 24.5482 7.75238 22.3306C9.88951 19.9923 12.7436 18.4761 15.3889 18.2737C15.4111 18.2716 15.4322 18.2684 15.4543 18.2642L15.4754 18.2599C16.709 18.1646 18.1228 18.2101 19.5662 18.3913C20.668 18.5301 21.7171 18.7388 22.7197 19.0185L22.9622 19.0662C23.2321 19.1001 23.4968 19.0546 23.7192 18.9423C23.8363 18.8861 23.9459 18.8098 24.0556 18.7081C24.0693 18.6954 24.0819 18.6827 24.0935 18.6689L24.4615 18.2525C24.4805 18.2324 24.4995 18.2112 24.5279 18.1783L24.7261 17.9431C24.7377 17.9293 24.7483 17.9145 24.7588 17.8997L24.7926 17.8509C24.9412 17.6359 25.0203 17.3837 25.0203 17.123C25.0203 16.5604 24.6629 16.0805 24.1041 15.8993L23.754 15.7912C23.7182 15.7785 23.6845 15.7679 23.6338 15.7541C22.3001 15.3452 20.8525 15.04 19.3321 14.8472C19.0812 14.8164 18.825 14.7878 18.5583 14.7624C19.4112 12.2058 21.3596 9.69256 23.987 7.78646C26.9898 5.60489 30.5028 4.43517 33.8778 4.5051C37.919 4.58244 41.5702 5.91533 44.1576 8.25584C44.7048 8.75064 45.2119 9.2751 45.7106 9.85996C45.9099 10.092 46.1471 10.2467 46.4613 10.3293C46.7575 10.3834 47.0559 10.3378 47.3132 10.2033C47.4102 10.1535 47.503 10.0888 47.6295 9.97333L47.7075 9.89599C47.7349 9.8695 47.7613 9.84301 47.7771 9.825L47.9479 9.63852C47.9742 9.61415 48.0038 9.58449 48.0417 9.54317L48.3106 9.23484C48.4171 9.1024 48.5605 8.88943 48.5847 8.56204C48.6047 8.27173 48.5215 7.98354 48.3496 7.74408C48.3348 7.72077 48.319 7.69852 48.3 7.67839L48.2621 7.63389C48.163 7.51205 48.0575 7.38914 47.9669 7.28848C44.497 3.02388 38.9112 0.407896 33.0122 0.290288C32.8772 0.288169 32.7433 0.287109 32.6083 0.287109C23.9164 0.287109 15.5661 6.73541 13.7811 14.7433C13.6082 14.7603 13.4437 14.7804 13.275 14.8037L13.2476 14.8111C9.7166 15.2954 6.49455 17.0383 3.9283 19.8503C1.31038 22.7152 0.0367432 26.2043 0.0367432 30.5166C0.0367432 39.2905 7.04702 46.1626 15.9952 46.1626C19.3142 46.1626 22.5131 45.065 25.2617 42.9756C25.9428 42.4172 26.7663 41.6406 27.7141 40.6626L40.6403 26.8326C41.9572 25.2804 42.9693 24.2876 43.5893 23.726L43.9108 23.4294C45.6484 21.8909 47.9331 21.0783 50.5194 21.0783C56.0631 21.0783 60.7465 25.9277 60.7465 31.6683C60.7465 34.2133 59.5677 37.0846 57.672 39.1634C55.9071 41.0949 53.2164 42.4161 50.6491 42.6122C50.6364 42.6132 50.6038 42.6185 50.5774 42.6238C49.4598 42.7118 48.1851 42.6704 46.8872 42.5051C45.912 42.3844 44.9673 42.1979 44.0405 41.9404L43.836 41.8991C43.8181 41.8959 43.8001 41.8927 43.7822 41.8917C43.5313 41.8673 43.2951 41.9097 43.0737 42.0199C42.9598 42.0771 42.8607 42.146 42.75 42.253L42.361 42.662C42.3262 42.6916 42.2966 42.7213 42.2713 42.7499L41.9972 43.0667C41.8401 43.2797 41.7526 43.5287 41.7442 43.7872C41.7262 44.3477 42.0647 44.8382 42.6172 45.0417L43.0842 45.2038C43.409 45.315 43.7464 45.3892 44.0711 45.457C44.9757 45.6837 45.9373 45.8628 46.9294 45.9899C47.3511 46.0429 47.7887 46.0853 48.2378 46.1203C48.6691 46.1478 49.1182 46.1658 49.5895 46.1754C49.7287 46.1775 49.8668 46.1785 50.0028 46.1785C50.3223 46.1785 50.6333 46.1722 50.9359 46.1616C51.1805 46.151 51.4356 46.1362 51.6939 46.1128C51.9407 46.0906 52.1789 46.0662 52.4541 46.0238C55.7795 45.5672 58.8729 43.9323 61.1672 41.4191C63.5521 38.8095 64.7118 35.633 64.7118 31.7096C64.7118 23.5756 58.195 16.9578 50.1841 16.9578Z"
                fill={4280908287}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
      </View>
    </View>
  </View>
</Modal>
`;

exports[`LoadingFullScreen component render correct renders well the loading next to the passed text 1`] = `
<Modal
  animationType="fade"
  hardwareAccelerated={false}
  testID="loading modal"
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(255, 255, 255, 0.75)",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 64,
          "justifyContent": "center",
          "position": "relative",
          "width": 64,
        }
      }
    >
      <View
        collapsable={false}
        style={
          Object {
            "borderBottomColor": "#2979FF",
            "borderLeftColor": "#2979FF",
            "borderRadius": 32,
            "borderRightColor": "#2979FF",
            "borderTopColor": "#D0D3E3",
            "borderWidth": 3.5,
            "height": 64,
            "position": "absolute",
            "transform": Array [
              Object {
                "rotateZ": "0deg",
              },
            ],
            "width": 64,
          }
        }
      />
      <View>
        <View
          height={25}
          width={36}
        >
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight={25}
            bbWidth={36}
            fill="none"
            focusable={false}
            height={25}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                Object {
                  "flex": 0,
                  "height": 25,
                  "width": 36,
                },
              ]
            }
            vbHeight={47}
            vbWidth={65}
            width={36}
            xmlns="http://www.w3.org/2000/svg"
          >
            <RNSVGGroup
              fill={null}
              propList={
                Array [
                  "fill",
                ]
              }
            >
              <RNSVGPath
                d="M50.1841 16.9578C46.4571 16.9578 43.6747 17.9474 41.1548 20.1724C40.9028 20.4012 40.6498 20.6386 40.3915 20.8812C40.1384 21.1217 39.8938 21.3622 39.6534 21.5974C39.6482 21.6027 39.6113 21.6388 39.607 21.6441L24.0672 38.2278C22.6459 39.7027 21.076 40.5842 20.6353 40.8152L20.455 40.9063C20.4466 40.9106 20.4381 40.9148 20.4297 40.919C18.9747 41.6194 17.5387 41.9605 16.0395 41.9605C9.66072 41.9605 4.47234 36.9045 4.47234 30.6882C4.47234 27.7512 5.7291 24.5482 7.75238 22.3306C9.88951 19.9923 12.7436 18.4761 15.3889 18.2737C15.4111 18.2716 15.4322 18.2684 15.4543 18.2642L15.4754 18.2599C16.709 18.1646 18.1228 18.2101 19.5662 18.3913C20.668 18.5301 21.7171 18.7388 22.7197 19.0185L22.9622 19.0662C23.2321 19.1001 23.4968 19.0546 23.7192 18.9423C23.8363 18.8861 23.9459 18.8098 24.0556 18.7081C24.0693 18.6954 24.0819 18.6827 24.0935 18.6689L24.4615 18.2525C24.4805 18.2324 24.4995 18.2112 24.5279 18.1783L24.7261 17.9431C24.7377 17.9293 24.7483 17.9145 24.7588 17.8997L24.7926 17.8509C24.9412 17.6359 25.0203 17.3837 25.0203 17.123C25.0203 16.5604 24.6629 16.0805 24.1041 15.8993L23.754 15.7912C23.7182 15.7785 23.6845 15.7679 23.6338 15.7541C22.3001 15.3452 20.8525 15.04 19.3321 14.8472C19.0812 14.8164 18.825 14.7878 18.5583 14.7624C19.4112 12.2058 21.3596 9.69256 23.987 7.78646C26.9898 5.60489 30.5028 4.43517 33.8778 4.5051C37.919 4.58244 41.5702 5.91533 44.1576 8.25584C44.7048 8.75064 45.2119 9.2751 45.7106 9.85996C45.9099 10.092 46.1471 10.2467 46.4613 10.3293C46.7575 10.3834 47.0559 10.3378 47.3132 10.2033C47.4102 10.1535 47.503 10.0888 47.6295 9.97333L47.7075 9.89599C47.7349 9.8695 47.7613 9.84301 47.7771 9.825L47.9479 9.63852C47.9742 9.61415 48.0038 9.58449 48.0417 9.54317L48.3106 9.23484C48.4171 9.1024 48.5605 8.88943 48.5847 8.56204C48.6047 8.27173 48.5215 7.98354 48.3496 7.74408C48.3348 7.72077 48.319 7.69852 48.3 7.67839L48.2621 7.63389C48.163 7.51205 48.0575 7.38914 47.9669 7.28848C44.497 3.02388 38.9112 0.407896 33.0122 0.290288C32.8772 0.288169 32.7433 0.287109 32.6083 0.287109C23.9164 0.287109 15.5661 6.73541 13.7811 14.7433C13.6082 14.7603 13.4437 14.7804 13.275 14.8037L13.2476 14.8111C9.7166 15.2954 6.49455 17.0383 3.9283 19.8503C1.31038 22.7152 0.0367432 26.2043 0.0367432 30.5166C0.0367432 39.2905 7.04702 46.1626 15.9952 46.1626C19.3142 46.1626 22.5131 45.065 25.2617 42.9756C25.9428 42.4172 26.7663 41.6406 27.7141 40.6626L40.6403 26.8326C41.9572 25.2804 42.9693 24.2876 43.5893 23.726L43.9108 23.4294C45.6484 21.8909 47.9331 21.0783 50.5194 21.0783C56.0631 21.0783 60.7465 25.9277 60.7465 31.6683C60.7465 34.2133 59.5677 37.0846 57.672 39.1634C55.9071 41.0949 53.2164 42.4161 50.6491 42.6122C50.6364 42.6132 50.6038 42.6185 50.5774 42.6238C49.4598 42.7118 48.1851 42.6704 46.8872 42.5051C45.912 42.3844 44.9673 42.1979 44.0405 41.9404L43.836 41.8991C43.8181 41.8959 43.8001 41.8927 43.7822 41.8917C43.5313 41.8673 43.2951 41.9097 43.0737 42.0199C42.9598 42.0771 42.8607 42.146 42.75 42.253L42.361 42.662C42.3262 42.6916 42.2966 42.7213 42.2713 42.7499L41.9972 43.0667C41.8401 43.2797 41.7526 43.5287 41.7442 43.7872C41.7262 44.3477 42.0647 44.8382 42.6172 45.0417L43.0842 45.2038C43.409 45.315 43.7464 45.3892 44.0711 45.457C44.9757 45.6837 45.9373 45.8628 46.9294 45.9899C47.3511 46.0429 47.7887 46.0853 48.2378 46.1203C48.6691 46.1478 49.1182 46.1658 49.5895 46.1754C49.7287 46.1775 49.8668 46.1785 50.0028 46.1785C50.3223 46.1785 50.6333 46.1722 50.9359 46.1616C51.1805 46.151 51.4356 46.1362 51.6939 46.1128C51.9407 46.0906 52.1789 46.0662 52.4541 46.0238C55.7795 45.5672 58.8729 43.9323 61.1672 41.4191C63.5521 38.8095 64.7118 35.633 64.7118 31.7096C64.7118 23.5756 58.195 16.9578 50.1841 16.9578Z"
                fill={4280908287}
                propList={
                  Array [
                    "fill",
                  ]
                }
              />
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "#585858",
          "fontFamily": "Roboto",
          "fontSize": 16,
          "fontWeight": "500",
          "lineHeight": 24,
          "marginTop": 25,
          "textAlign": "center",
          "width": "50%",
        }
      }
    >
      Lorem ipsum
    </Text>
  </View>
</Modal>
`;
